src/tabs/tab-header-bar.tsx

```diff
@@ -194,13 +201,18 @@ export function TabHeaderBar({
     function isElementRegistered(element: HTMLElement) {
       return navigationAPI.current?.isRegistered(element) ?? false;
     }
-    function isElementDisabled(element: HTMLElement) {
+    function isElementFocusable(element: HTMLElement) {
       if (element instanceof HTMLButtonElement) {
         return element.disabled && element.getAttribute('aria-selected') !== 'true';
       }
+      if (element instanceof HTMLAnchorElement) {
+        // aria-describedby is present only if the element is disabled with reason. in this case it should be focusable, otherwise no
+        return element.getAttribute('aria-disabled') === 'true' && !element.hasAttribute('aria-describedby');
```

Comment:
Would it make sense to add some `tab-focusable` style to focusable tabs and simplify the check here?

Reply:
updated